docs/faq.md

```diff
@@ -0,0 +1,35 @@
+# AKS-Engine FAQ
+
+This page provides help with the most common questions about AKS-Engine.
+
+### What's the Difference Between AKS and AKS-Engine?
+
+Azure Kubernetes Service ([AKS][]) is a Microsoft Azure service that supports fully managed Kubernetes clusters. [AKS-Engine][] is an Azure open source project that creates Kubernetes clusters with your custom requirements. AKS uses AKS-Engine internally, but they are not the same.
+
+AKS clusters can be created in the Azure portal or with `az aks create` in the [Azure command-line tool][]. AKS-Engine clusters can be created with `aks-engine deploy` in the AKS-Engine command-line tool, or by generating the ARM templates with `aks-engine generate` and deploying them as a separate step.
+
+### What's the Difference Between `acs-engine` and `aks-engine`?
+
+AKS-Engine is the next version of the ACS-Engine project. AKS-Engine supports current and future versions of [Kubernetes][], while ACS-Engine also supported the Docker Swarm and Mesos DC/OS container orchestrators.
+
+### Can I Scale or Upgrade an `acs-engine` Cluster with `aks-engine`?
```

Comment:
are the upper cases for Scale, Upgrade and Cluster intentional?

Reply:
Oh I guess all the questions are written that way

Reply:
I prefer using Title Case for headlines, but that's because I used to be a newspaper editor in a previous life. If it looks odd, I'm happy to change it.